src/lib/types.ts

```diff
@@ -11,3 +11,8 @@ export type DeepPick<T, K extends string> = T extends object
         : DeepPick<T[P], Tail<Extract<K, `${P}.${string}`>>>
     }
   : T
+
+// Makes an object's optional property required
+export type WithRequiredProperty<T, K extends keyof T> = T & {
```

Comment:
Because `Omit<T, 'property'> & {property: SomeType}` is too boilerplaty, might as well

Reply:
Actually yeah, let's not add an additional type - 241731470135fc99c561169a8b4f59cdb7a8f378 🥖